package-builder/build-manifest.nix

```
@@ -25,7 +25,7 @@ let
   dollar_out_error_and_exit = ''
     echo "❌ ERROR: Build command did not copy outputs to '\$out'." 1>&2
     echo "- copy a single file with 'cp bin \$out'" 1>&2
```

Can we fix this while we're in the neighborhood?

```suggestion
    echo "- copy a single file with 'mkdir -p $out/bin && cp file $out/bin'" 1>&2
    echo "- copy a bin directory with 'mkdir $out && cp -r bin $out'" 1>&2
```

---

I think the \$out must stay escaped because out will be different in pure builds especially when you change the script to contain the path $out evaluates to 


But the rest we can update for sure 👍